### PR TITLE
DOCSP-30768 Adds SRV connection option

### PR DESCRIPTION
**Pull request by kennethdyer**
## Description

Adds SRV option to mongosync connection strings.

## Staging

* [Connection String](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-30768-srv-connection/connecting/atlas-to-atlas/#connection-strings)

## Jira
* [DOCSP-30768](https://jira.mongodb.org/browse/DOCSP-30768)

## Build
* [Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64e3e4cc57e37806c50d07e2)
* [Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64e4d48257e37806c53f36e6)
* [Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64f0ee1d24fcc731b435cade)

### CODE REVIEW

File: source/includes/fact-connection-strings.rst [hunk 1]
```diff
@@ -1,11 +1,17 @@
-:ref:`mongosync <c2c-mongosync>` uses a standard :ref:`MongoDB URI
-connection string <mongodb-uri>` to connect clusters. 
+:ref:`mongosync <c2c-mongosync>` uses a :ref:`MongoDB URI
+connection string <mongodb-uri>` to connect clusters:
 
-The standard URI connection scheme has the form:
+- The SRV connection scheme has the form:
```
Comment (ajhuh-mdb) [inline source/includes/fact-connection-strings.rst:4]: [question] The JIRA ticket mentions that we could "also note that you can grab the SRV string from the Atlas UI: https://www.mongodb.com/docs/atlas/tutorial/connect-to-your-cluster-v2/" 

Should we note this somewhere—maybe under the code blocks?

**Review by ajhuh-mdb — APPROVED** [review]
LGTM % one question. Thanks for making these updates!

File: source/includes/fact-connection-strings.rst (hunk 1 above)
Comment (kennethdyer) [reply source/includes/fact-connection-strings.rst:4]: Fixed

**Review by marieatterbury — APPROVED** [review]
lgtm

**Pull request merged**